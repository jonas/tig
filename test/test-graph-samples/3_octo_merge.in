commit A B C D
    Commit A - Merges B, C, and D
commit C E
    Commit C after E
commit E D B
    Commit E - Merges D and B
commit B D
    Commit B after D
commit D
    Commit D
