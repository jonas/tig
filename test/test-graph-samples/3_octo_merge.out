●─┬─╮ Commit A - Merges B, C, and D
│ ● │ Commit C after E
│ ●─│─╮ Commit E - Merges D and B
●─│─│─╯ Commit B after D
◎─┴─╯ Commit D
